assets/src/domain/eventEditor/services/context/EntityListContext/DatetimesListContext.tsx

```diff
@@ -21,13 +22,29 @@ export const DatetimesListProvider: React.FC = ({ children }) => {
 	// memoize filter state
 	const filterState = useMemo(() => filters, [filtersStr]);
 
-	const filteredEntities = useFilteredEntities(domain, datesList, datetimes, filterState);
+	const { setSortBy, sortingEnabled } = filterState;
+
+	let filteredEntities = useFilteredEntities(domain, datesList, datetimes, filterState);
+
+	if (filterState.sortingEnabled) {
+		filteredEntities = notTrashed(filteredEntities);
+	}
 
 	// Update Edtr state for isChained filter
 	const { setVisibleDatetimeIds } = useEdtrState();
 	useEffect(() => {
-		setVisibleDatetimeIds(getGuids(filteredEntities));
-	}, [filteredEntities]);
+		// update only when not sorting
+		if (!sortingEnabled) {
+			setVisibleDatetimeIds(getGuids(filteredEntities));
+		}
+	}, [filteredEntities, sortingEnabled]);
```

Comment:
maybe use `assets\src\application\services\utilities\memo\entityListCacheIdString.ts` here instead of `filteredEntities`???

Reply:
Done